packages/calypso-products/src/experiments.ts

```diff
@@ -22,3 +22,13 @@ export const setPlansListExperiment = ( experimentName: string, variation: strin
 export const getPlansListExperiment = ( experimentName: string ): string | undefined => {
 	return getExperiment( PLANS_LIST_NAMESPACE, experimentName );
 };
+
+export const setTrailMapExperiment = ( variation: string ): void => {
+	setExperiment( PLANS_LIST_NAMESPACE, 'wpcom_trail_map_feature_structure_experiment', variation );
```

Comment:
If we mean to reuse the same namespace, we can just reuse `setPlansListExperiment` and `getPlansListExperiment` defined above.